src/field.cpp

```diff
@@ -719,12 +716,13 @@ bool map::process_fields_in_submap( submap *const current_submap,
                                     time_added = -rng(80 * vol, 300 * vol);
                                     smoke++;
                                 }
+                                // burn_amt will get multiplied by stack size in item::burn
                                 burn_amt = cur->getFieldDensity();
 
                             } else if( fuel->made_of("powder") ) {
-                                // Any powder will fuel the fire as much as its volume
+                                // Any powder will fuel the fire as 100 times much as its volume
                                 // but be immediately destroyed.
-                                time_added = vol;
+                                time_added = vol * 100;
                                 destroyed = true;
```

Comment:
Yeah, OK, so dropping a ludicrous amount of powder on a fireplace means it keeps going way longer than expected.  So noted.